contents/blog/how-we-built-email-onboarding.md

```diff
@@ -0,0 +1,132 @@
+
+By Q3 2023, we were adding more and more new products to PostHog. We wanted to feature these in our flow, but we were also worried about becoming spammy. Thankfully, now that we had Customer.io in place, we were able to judge this based on the data. 
+
+![Onboarding 3.0](../images/blog/onboarding/onboarding_3.png)
+<caption>New emails for 3.0 tried to emphasize what was possible with PostHog, directing users to helpful blog content</caption>
+
+Our conversion event was a user log-in within one week of opening an email. We hadn’t yet thought of anything better. 
+
+- **Open rate:** 56%
+- **Click-through-rate:** 7.4%
+- **Conversion rate:** 3%
+- **Unsubscribe rate:** 0.6%
+
+Based on this high open rate and low unsubscribe rate we felt that users didn't mind getting the emails, and the click-through rate suggested they liked what they read. So, we continued adding more messages. 
+
+> **What does good look like for an email onboarding flow?**
+> Benchmarks vary wildly by industry, implementation, and product. As a rule though, the guideline benchmarks I use from my experience in other startups are:
+>
+> - **Open rate:** Anything above 40% is OK. 50% is the goal.
+> - **Click-through-rate:** Anything above 4% is OK. 6% is the goal. 
+> - **Conversion rate:** Anything above 3% is OK. 5% is the goal. 
+> - **Unsubscribe rate:** Anything below 1% is OK. 0% is the goal.
+
+## Onboarding 3.1: The one where we added personalization
+
+Data is important, but I strongly believe it doesn’t tell the whole story. You also need feedback and intuition. Most of the feedback on our emails came in [3.1](https://github.com/PostHog/meta/issues/127), when we started sending personalized recommendations to users from [my personal email](mailto:joe@posthog.com).
+
+How this works is simple: when a user signs up, they can optionally tell us what their role is. If their `role_at_organization` = `engineer` we trigger an email from me telling them about [our Product for Engineers newsletter](https://newsletter.posthog.com/). 
+
+![Onboarding 3.1](../images/blog/onboarding/onboarding_31.png)
+<caption>Onboarding 3.1 was when we really started tailoring emails based on what we knew about users</caption>
+
+This email performed very well, with a 68% open rate and a 16% CTR. Two emails into the flow and developers were already opting in to _more_ emails! 
+
+Best of all though, because it was personalized and came from me directly, it also earned a steady trickle of replies. I responded in kind and was able to feed in further improvements, including adding another option for `role_at_organization` = `founder`. 
+
+## Onboarding 3.3: The one where we added experiments
+
+In [3.3](https://github.com/PostHog/meta/issues/140) we started running a series of tests and experiments. We added and then removed a `founder` targeted email, which invited teams to join [PostHog for Startups](/startups), because it was _too_ successful and cannibalised revenue.
```

Comment:
```suggestion
In [3.3](https://github.com/PostHog/meta/issues/140) we started running a series of tests and experiments. We  removed the `founder` targeted email, which invited teams to join [PostHog for Startups](/startups), because it was _too_ successful and cannibalised revenue.
```